### ISSUE TITLE
v5.5: RQlog: Error in logline 34 - request log writing failure

### ISSUE THREAD

**Issue by opoplawski** [e2guardian/e2guardian#786]
Using 5.5.4r - I'm enabling the request log, but it's failing with:

```
e2guardian[253669]: RQlog: Error in logline 34 0

                                                                  0
                                                                  0
                                                                  0
                                                                  https://www.google.com
                                                                  hw0: MITM_MITM
                                                                  GET
                                                                  -
                                                                  10.10.20.7
                                                                  80
                                                                  0
                                                                  0
                                                                  0
                                                                  0
                                                                  0
                                                                  0
                                                                  0
                                                                  0
                                                                  0

                                                                  1694013275
                                                                  374429
                                                                  1694013275
                                                                  374429




                                                                  0
                                                                  0
                                                                  8080:PM::def
```

I think these are the relevant config options:

```
logfileformat = 4
logsslerrors = on
set_requestlog = 'file:/var/log/e2guardian/request.log'
usedashforblank = on
```

**Comment by philipianpearce** [comment]
Thanks for this report - fixed in latest v5.5

**Comment by opoplawski** [comment]
Fix looks good to me, thanks!
